src/earn/EarnActivePool.tsx

```diff
@@ -33,27 +35,39 @@ function PoolDetailsLoading() {
   )
 }
 
-export default function EarnActivePool() {
+interface Props {
+  cta: 'ExitAndDeposit' | 'ViewPools'
+  depositTokenId: string
+  poolTokenId: string
+}
+
+export default function EarnActivePool({ depositTokenId, poolTokenId, cta }: Props) {
   const { t } = useTranslation()
-  const token = useTokenInfo(networkConfig.arbUsdcTokenId)
-  const poolToken = useTokenInfo(networkConfig.aaveArbUsdcTokenId)
-  const asyncPoolInfo = useAsync(async () => {
-    if (!token || !token.address) {
-      // should never happen
-      Logger.warn(TAG, `Token with id ${networkConfig.arbUsdcTokenId} not found`)
-      throw new Error(`Token with id ${networkConfig.arbUsdcTokenId} not found`)
-    }
-    if (!isAddress(token.address)) {
-      Logger.warn(
-        TAG,
-        `Token with id ${networkConfig.arbUsdcTokenId} does not contain a valid address`
-      )
-      throw new Error(
-        `Token with id ${networkConfig.arbUsdcTokenId} does not contain a valid address`
-      )
+  const depositToken = useTokenInfo(depositTokenId)
+  const poolToken = useTokenInfo(poolTokenId)
+  const asyncPoolInfo = useAsync(
+    async () => {
+      if (!depositToken || !depositToken.address) {
+        throw new Error(`Token with id ${depositTokenId} not found`)
+      }
+
+      if (!isAddress(depositToken.address)) {
+        throw new Error(`Token with id ${depositTokenId} does not contain a valid address`)
+      }
```

Comment:
I guess these never should happen, but if it does, do these get logged somewhere?

Reply:
They'll get logged in the `onError` of the `useAsync` hook.